framework/source/class/qx/ui/basic/Label.js

```diff
@@ -535,7 +535,12 @@ qx.Class.define("qx.ui.basic.Label",
     _applyValue : function(value, old)
     {
       // Sync with content element
-      this.getContentElement().setValue(value);
+      if (value.translate) {
```

Comment:
Does this mean, that if I have something like
```javascript
value = this.tr('the real value');
```
that then value isn't a string anymore?!?!?